Navaids/FIXES_Old.txt

```diff
@@ -19,3 +19,11 @@ MAPLE N052.23.36.000 W001.40.51.000
 OLIVE N052.24.12.000 W001.56.12.000
 TAWNY N051.38.46.000 E000.09.06.000
 
+;withdrawn 12/2021
+CAMBO N049.17.30.000 W005.51.35.000
+OGAGI N049.27.22.000 W005.01.57.000
+MOSIS N049.00.26.000 W007.12.53.000
+PHILI N049.28.31.000 W007.01.17.000
+PIKOD N049.24.38.000 W005.15.59.000
+SUPAP N049.26.02.000 W005.49.22.000
+XAVAP N049.14.02.000 W006.08.35.000
```

Comment:
Looks like PEMAK Is missing

Reply:
good spot was going cross-eyed looking at so many fixes

Reply:
Lots of numbers would hurt my eyes